addons/account/views/audit_trail_report_views.xml

```diff
@@ -27,7 +27,21 @@
                 <field name="account_audit_log_move_id"/>
                 <field name="author_id"/>
                 <field name="date" string="Date"/>
-                <filter string="Update Only" name="update_only" domain="[('tracking_value_ids', '!=', False)]" groups="base.group_system"/>
+                <filter string="Updated Entries" name="audit_updated_entries" domain="[('tracking_value_ids', '!=', False)]" groups="base.group_system"/>
+                <filter string="Cancelled"
+                        name="audit_cancelled"
+                        domain="[('tracking_value_ids.field_id.name', '=', 'state'),
```

Comment:
Would probably be better if possible to filter based on real field id: indeed we have several "state" tracked fields, less "field state on account.move" (don't remember how xml ids are generated for fields). Would it be possible to use a ref here and have the real ID ?

Reply:
I understand your point but I can't find any example of using ref in xml domains. I suppose the alternative would be to create a invisible field just for the ids in domain which will be less ideal